src/schemas/__tests__/Entity.test.js

```diff
@@ -147,4 +147,13 @@ describe(`${schema.Entity.name} denormalization`, () => {
 
     expect(denormalize(1, menuSchema, entities)).toMatchSnapshot();
   });
+  it('can denormalize previously normalized data', () => {
```

Comment:
Entity doesn't use the `Polymorphic` schema denormalization. This test will pass regardless of the change you made. Looks like an oversight on my part to not have included an Entity test with a custom schemaAttribute, but that's mostly because it doesn't matter—the lookup for denormalize goes by key and doesn't dig into the entity to find the ID.